core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -74,6 +75,7 @@
 import io.reactivex.android.schedulers.AndroidSchedulers;
 import java.io.File;
 import java.io.IOException;
+import java.lang.reflect.Field;
```

Comment:
Unused import, if you go to File>Settings (ctrl+alt+s) in intellij and toggle some settings it should stop this from happening. Here is my setup:
![image](https://user-images.githubusercontent.com/3358282/72161632-62a1ee80-33b8-11ea-83fc-cd680f1c8f23.png)

Reply:
Done!